app/routes/surveys/components/SurveyList/SurveyItem.tsx

```diff
@@ -36,7 +37,7 @@ const SurveyItem = ({ survey }: Props) => {
           <div>
             <div className={styles.surveyTime}>
               For arrangement{' '}
-              <Link to={`/events/${event.id}`}>{event.title}</Link>
+              <Link to={`/events/${event?.id}`}>{event?.title}</Link>
```

Comment:
Kinda useless to only show "For arrangement" if event is undefined

Reply:
Yeah, I think all surveys have an event. But it will probably look a bit weird while loading:/